google-cloud-bigquery/lib/google/cloud/bigquery/table.rb

```diff
@@ -118,6 +118,35 @@ def table_ref
           table_ref
         end
 
+        ###
+        # Is the table partitioned?
+        #
+        # @!group Attributes
+        #
+        def time_partitioning?
+          !@gapi.time_partitioning.nil?
+        end
+
+        ###
+        # The period for which the table is partitioned, if any.
+        #
+        # @!group Attributes
+        #
+        def time_partitioning_type
+          ensure_full_data!
+          time_partitioning.type if time_partitioning?
```

Comment:
`time_partitioning.type` needs to be changed to `@gapi.time_partitioning.type`.

I get:

```sh
  3) Error:
Google::Cloud::Bigquery::Table::bigquery#test_0004_allow tables to be created with time_partioning enabled:
NameError: undefined local variable or method `time_partitioning' for #<Google::Cloud::Bigquery::Table:0x007f8299e6da20>
Did you mean?  time_partitioning?
               time_partitioning_type
    google-cloud-bigquery/lib/google/cloud/bigquery/table.rb:137:in `time_partitioning_type'
    google-cloud-bigquery/acceptance/bigquery/table_test.rb:139:in `block (2 levels) in <top (required)>'